packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -120,6 +120,68 @@ lambdaAction.outputArtifact('Out2'); // returns the named output Artifact, or th
 See [the AWS documentation](https://docs.aws.amazon.com/codepipeline/latest/userguide/actions-invoke-lambda-function.html)
 on how to write a Lambda function invoked from CodePipeline.
 
+### Lambda deployed through CodePipeline
+
+If you want to deploy your Lambda through CodePipeline,
+you can use the special `Code` convenience method, `lambda.Code.codePipeline`.
+Note that your Lambda must be in a separate Stack than your Pipeline.
+The Lambda itself will be deployed, alongside the entire Stack it belongs to,
+using a CloudFormation CodePipeline Action. Example:
+
+```typescript
+const lambdaCode = lambda.Code.codePipeline(); 
+const lambda = new lambda.Function(lambdaStack, 'Lambda', {
+  code: lambdaCode,
+  handler: 'index.handler',
+  runtime: lambda.Runtime.NodeJS810,
+});
+
+// other resources that your Lambda needs, added to the lambdaStack...
+
+const pipeline = new codepipeline.Pipeline(pipelineStack, 'Pipeline');
+
+// add the source code repository containing this code to your Pipeline,
+// and the source code of the Lambda Function, if they're separate
+pipeline.addStage({
+  name: 'Source',
+  actions: [
+    // ...
+  ],
+});
+
+// add a build Action to your Pipeline,
+// that calls `cdk synth` on the lambdaStack,
+// and saves it to some file,
+// and a separate build for your Lambda source code - if needed
+pipeline.addStage({
+  name: 'Build',
+  actions: [
+    lambdaBuildAction,
+    cdkBuildAction,
+  ],
+});
+
+// finally, deploy your Lambda code
+const parameterOverrides: { [name: string]: any } = {};
+parameterOverrides[lambdaCode.bucketNameParam.logicalId] = lambdaBuildAction.outputArtifact.bucketName;
+parameterOverrides[lambdaCode.bucketKeyParam.logicalId] = lambdaBuildAction.outputArtifact.objectKey;
```

Comment:
This doesn't seem like a good experience.

Reply:
Changed to be just `parameterOverrides: lambdaBuildAction.outputArtifact.fillParameters(lambdaCode)`.